CHANGELOG.md

```diff
@@ -0,0 +1,72 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
You might want to add some quick notes on how to migrate from rllab

Reply:
Isn't it a better idea to have an independent migration guide? Since if someone wants to migrate, https://rlgarage.readthedocs.org/en/latest/ will be his/her first place to look into, and people may easily fail to notice the presence of a CHANGELOG.

Reply:
I agree with @naeioi, but I think a separate issue is required for this.